trefoil/netcdf/tests/test_crs.py

```diff
@@ -56,7 +55,6 @@ def test_get_crs(tmpdir):
 
     out_data = CRS.from_string(out_proj4).to_dict()
 
-    assert len(out_data) == 4  # There should be 4 parameters

```

Comment:
This assertion failed, but it's hard to know what 4 things were wanted. More direct assertions of the contents of `out_data` follow.